src/docs/concepts-users-and-authentication/user-permissions/index.md

```diff
@@ -98,12 +110,160 @@ Now, you can check or uncheck the permissions checkbox for this user.
+
+Read more about this feature in the
+[Help Center](https://www.sharetribe.com/help/en/articles/9790336).
+
+## API level restrictions
+
+Toggling permissions under the "Access Control" tab in the Console
+directly impacts API behavior. When permissions are modified, endpoint
+access is adjusted accordingly, ensuring that users cannot bypass the
+restrictions imposed by these settings. This guarantees access control
+policies are strictly enforced. These restrictions only apply in the
+Marketplace API.
+
+### Making marketplace private
+
+Toggling this setting in Console requires users to log in to access
+listings.
+
+Making the marketplace private from Console will restrict the following
+endpoints:
+
+- `GET /users/show`
+- `GET /listings/query`
+- `GET /listings/show`
+- `GET /timeslots/query`
+- `GET /reviews/query`
+- `GET /reviews/show`
+- `GET /sitemap_data/query_listings`
+
+When this setting is toggeld, these endpoints will return a 403
+Forbidden response, indicating that access is denied due to the
+marketplace's private status.
+
+### Approve users who want to join
+
+Toggling this setting in Console will require users to be approved by an
+operator to join the marketplace.
+
+This will restrict the following endpoints:
+
+Listings:
+
+- `POST /own_listings/create`
+- `POST /own_listings/create_draft`
+- `POST /own_listings/publish_draft`
+- `POST /own_listings/discard_draft`
+- `POST /own_listings/open`
+- `POST /own_listings/close`
+- `POST /own_listings/update`
+- `POST /own_listings/add_image`
+
+Transactions:
+
+- `POST /transactions/initiate`
+- `POST /transactions/initiate_speculative`
+- `POST /transactions/transition`
+- `POST /transactions/transition_speculative`
+
+Availability exceptions:
+
+- `POST /availability_exceptions/create`
+- `POST /availability_exceptions/delete`
+
+Stock adjustments:
+
+- `POST /stock_adjustments/create`
+- `POST /stock_adjustments/compare_and_set`
+
+When this setting is toggled, these endpoints will return a 403
+Forbidden response, indicating that access is denied because the user
+has not been approved.
+
+### Restrict publishing rights
+
+Toggling this setting in the Console allows you to manually select which
+users can publish listings.
+
+- `POST /own_listings/create_draft`
+- `POST /own_listings/publish_draft`
+- `POST /own_listings/create`
+- `POST /own_listings/open`
+
+These endpoints will return a 403 Forbidden response if the user does
+not have publishing rights.
+
+### Restrict transaction rights
+
+Toggling this setting in Console allows you to manually select which
+users can initiate transactions.
+
+When transaction rights have been revoked for a marketplace user, the
+following Marketplace API endpoint will be restricted:
+
+- `POST transactions/initiate`
+
+This endpoint will return a 403 Forbidden response if the user doesn't
+have permission to initiate transactions. Note that users can still
+transact any existing transactions they've initiated.
```

Comment:
Maybe a mention here that since transactions/transition is not restricted, it depends on their client app whether users can transact existing transactions – web template blocks e.g. checkout page when coming from TransactionPage in inquiry state, so making a payment on an existing transaction is prevented for non-transaction rights users as well.